src/components/FilteredPeople.tsx

```diff
@@ -0,0 +1,55 @@
+import { SortType } from '../types/SortType';
+import { Person } from '../types/Person';
+
+export const FilteredPeople = (
```

Comment:
FilteredPeople function has too many parameters, consider using an `options` parameter to group them together.

Reply:
also, use camelCase for not component functions and add a verb that explains what the function does, i.e `getFilteredPeople` or `filterPeople`

Reply:
fixed